models/orogen/skid4_control.rb

```diff
@@ -1,17 +1,19 @@
+require 'models/services/motion2d_control_loop'
+require 'models/services/joints_control_loop'
 require 'models/blueprints/control'
 
 class OroGen::Skid4Control::SimpleController
-    provides Base::ActuatorControllerSrv, :as => 'controller'
-    provides Base::Motion2DControlledSystemSrv, :as => 'controlled_system'
+    provides Rock::Services::JointsOpenLoopController, :as => 'controller'
```

Comment:
What do you mean with :as .. syntax? Is not that the as: syntax? 

Uncommenting the lines below was a mistake. I did not intended to push this. The problem is that I am not sure how to fix this because using_task_library 'controldev' breaks on my system. 

Either control_dev must be added to the manifest or should be removed. And I would be in favor of removing it because it is a bit to specific.